scripts/hist_importer.py

```diff
@@ -142,7 +142,8 @@ def run():
     source, dest = args.source, args.dest
     query = {
         'firefox': 'select url,title,last_visit_date/1000000 as date '
-                   'from moz_places',
+                   'from moz_places where url like "http%" or url '
+                   'like "ftp%" or url like "file://%"',
```

Comment:
I would prefer a black-list over a white-list here, what if people want to import other records like Magnet URIs?

Reply:
IMHO the blacklist could be infinite (AFAIK every extension can create its own). Instead, we know which protocols qutebrowser handles.